src/common/copilot/assets/scripts/copilot.js

```diff
@@ -127,7 +127,7 @@
     copyButton.innerHTML = clipboardSvg;
     copyButton.classList.add("action-button");
     copyButton.classList.add("copy-button");
-    copyButton.title = "Copy to clipboard";
+    copyButton.title = copilotStrings. COPY_TO_CLIPBOARD_MESSAGE;
```

Comment:
breaking typo - space in `copilotStrings. COPY_TO_CLIPBOARD_MESSAGE;`

Reply:
Thanks! fixed